kaggle_workshop/Student Exam Performance Workshop.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<p>I'd prefer to avoid a discussion of `iloc` and `loc` at this stage, perhaps it would be simpler to create a new column using `apply`? That would also help with our function discussion. E.g. the solution would be:</p><p><br></p><p>```python</p><p>def degree_status(edu):</p><p>    if edu in {'high school', 'some high school'}:</p><p>        return 'no_degree'</p><p>    return 'has_degree'</p><p><br></p><p>df['degree_binary'] = df['parentDegree'].apply(degree_status)</p><p>```</p>

 _Reply via <a href='https://app.reviewnb.com/georgetown-analytics/XBUS-501-01.Software-Engineering-for-Data/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Also - this is just stub code, I didn't test that it would work!

Reply:
Thanks for the tip! I agree that this will likely be easier to understand and does fit in better with the idea of writing reusable code. I updated the notebook to contain your function (which I tested and works perfectly). Students will just need to complete the function to return the proper string.

Thanks again!

Reply:
Awesome, thank you! Will merge now.